testing-unit-common/src/main/java/com/bumble/appyx/testing/unit/common/util/TestUpNavigationHandler.kt

```diff
@@ -0,0 +1,21 @@
+package com.bumble.appyx.testing.unit.common.util
+
+import com.bumble.appyx.core.routing.upnavigation.UpNavigationHandler
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class TestUpNavigationHandler : UpNavigationHandler {
+    var invoked: Boolean = false
+
+    override fun handleUpNavigation() {
+        invoked = true
+    }
+
+    fun assertInvoked() {
+        assertTrue(invoked, "FallbackUpNavigationHandler.handle was not invoked")
```

Comment:
Change message please.

Reply:
Ops, didn't see it
Changed in c93826e0ceaa055865a0ec5581284845e1f2706f